datanode/gateway/graphql/schema.graphql

```diff
@@ -2006,9 +2006,6 @@ type Market {
   "Risk factors for the market"
   riskFactors: RiskFactor
 
-  "Liquidity Provision order price range"
-  lpPriceRange: String!
-
```

Comment:
This shouldn't be removed so that it can still be compatible rather than a breaking change.